contents/docs/user-guides/sessions.md

```diff
@@ -4,4 +4,75 @@ sidebar: Docs
+* **Can a session span browser tabs and windows?:** Yes, as a user moves between different tabs and windows, their events will be grouped into a single session. However, if they open a completely new browser (e.g. move from Chrome to Firefox) then a new session will be created.
+* **Which clients support sessions?:** Today, PostHog-JS is the only client that supports sessions. However, we are planning to add it to the other front-end libraries.
+* **How are sessions split?:** Sessions are split when there is no user activity for more than 30 minutes.
+* **What defines user activity for a session?:** User activity is defined as any event sent from the client (including Autocapture events). Also, if recordings are enabled, then user activity from the recording (like mouse movement) will extend the session.
+* **Is there a limit on session length?:** Yes, the longest allowed session is 24 hours. If a session is longer than that, it will be split.
+* **Is a session the same as a recording?:** Both sessions and recordings are tracked using the same ID. So if a new session is created, then a new recording is created too.
+
+## Session analysis
+
+Within the trends tab of insights, you can analyze sessions along with your other data. 
+
+<blockquote class='warning-note'>
+<b>IMPORTANT: </b> 
+When performing session analysis, PostHog will only include events that have been been tagged with a session. Right now, that is only posthog-js events sent from version <code>1.16.1</code> (released Nov. 7 2021) and later.
+</blockquote>
+
+### Session counts
+
+Within the trends tab, you can aggregate results by the count of unique sessions. This provides the count of unique sessions where the specified event occurred. If you have specific events that are sent a bunch of times within a single session, this can be helpful to remove the noise.
+
+To do this analysis, simply select Unique sessions from the aggregation dropdown.
+
+
+![Session counts](../../images/user-guides/sessions/unique-sessions.png)
+
+### Using session counts in formulas
+
+Combining session counts with other trends is a very powerful way to understand things like:
+* The average number of sessions per user where a specific action occurs
+* The average number of a specific events per session
+
+To do this analysis, simply make the series in your trends (e.g. unique sessions where an event occurs), and then use that series in a formula.
+
+![Session counts](../../images/user-guides/sessions/session-formula.png)
+
+### Filtering by session duration
+
+Sometimes, it can be helpful to filter out sessions that are either too short or too long. For example, this allows you to dig into users who only use your product for a few seconds, or filter those users out, so you can better understand those who stick around longer.
```

Comment:
```suggestion
Sometimes, it can be helpful to filter out sessions that are either too short or too long. For example, this enables you to dig into users who only use your product for a few seconds, or filter those users out, so you can better understand those who stick around longer.
```